### DISCUSSION TITLE
Optimized Trend-following Indicator with Probability and Distribution-adjusted EMA and Extremum Analysis

### DISCUSSION THREAD

**Discussion by ansarizafar** [bludnic/opentrader#62] [Ideas]
Could you please add a strategy / indicators which is described in this article  https://medium.com/superalgos/optimized-trend-following-indicator-with-probability-and-distribution-adjusted-ema-and-extremum-7c8252f00ff0

**Comment by bludnic** [comment]
Hey @ansarizafar, thanks for your interest in the project!

I could, but it's not a priority right now. Currently, I'm focused on building DCA and ARB strategies. I'm also working on a Wiki that explains how to build a custom strategy, so if you have programming skills, you'll be able to create it by yourself.

**Reply by ansarizafar** [reply]
Yes I have programming skills but I am new to trading so I would need your help. Have you seen this project https://superalgos.org/

**Reply by bludnic** [reply]
Yep, I've heard about this project. From the article you shared, as I understood, they have already implemented that strategy, so why not use their bot?

**Reply by ansarizafar** [reply]
Superalgos is very complex and the project is dead. Last release was in Feb.
